cocos/platform/android/CCFileUtils-android.cpp

```diff
@@ -249,6 +268,12 @@ FileUtils::Status FileUtilsAndroid::getContents(const std::string& filename, Res
     } else {
         relativePath = fullPath;
     }
+    
+    if (obbfile)
+    {
+        if (obbfile->getFileData(relativePath, buffer))
+            return FileUtils::Status::OK;
```

Comment:
I think you misread it, this code is in FileUtilsAndroid::getContents which returns a FileUtils::Status